src/Configuration/ConfigDefinition.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\SynapseTransformation\Configuration;
+
+use Keboola\Component\Config\BaseConfigDefinition;
+use Symfony\Component\Config\Definition\Builder\ArrayNodeDefinition;
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+
+class ConfigDefinition extends BaseConfigDefinition
+{
+    protected function getRootDefinition(TreeBuilder $treeBuilder): ArrayNodeDefinition
+    {
+        $rootNode = parent::getRootDefinition($treeBuilder);
+        $rootNode->children()->append($this->getAuthorizationDefinition());
+        return $rootNode;
+    }
+
+    protected function getAuthorizationDefinition(): ArrayNodeDefinition
+    {
+        $authorizationNode = (new TreeBuilder('authorization'))->getRootNode();
+        assert($authorizationNode instanceof ArrayNodeDefinition);
+
+        // @formatter:off
+        $authorizationNode
+            ->isRequired()
+            ->children()
+                ->arrayNode('workspace')
+                    ->isRequired()
+                    ->children()
+                        ->scalarNode('host')
+                            ->isRequired()
+                        ->end()
+                        ->integerNode('port')
+                            ->isRequired()
```

Comment:
jo jeste tohle - taky kdyztak over s connection tym jestli budou u synapse workspace generovat port, jestli ne, tak musi byt optional

Reply:
ok. zistim.

Reply:
dohodnute, ze port bude optional s default value:
slack: https://keboola.slack.com/archives/CFVRE56UA/p1584350092034200?thread_ts=1584109516.023700&cid=CFVRE56UA
commit: https://github.com/keboola/synapse-transformation/pull/1/commits/f1aec5bf87caccdbaaaf83e8581c26b29a6becb5

Reply:
ok. upravene v https://github.com/keboola/synapse-transformation/pull/1/commits/5b9f9ca3745c418670b07b73628c4bba35cbed4b

Boli tam 2 issues:
- ja som mal typ NUMERIC aj tam kde malo byt FLOAT
- db reportuje `length` aj pre typy, ktore ju maju staticku (napr. `bigint` 8 bajtov), ... fixol som to  v kode aj v testoch